src/data/analytics.js

```diff
@@ -1,21 +1,24 @@
+// TODO: rename this in cloudflare
```

Comment:
Remember this TODO. Do you have a new value for it?

Reply:
Not sure who has access to our gtag id so I've changed it back to analytics for now.

Reply:
I think a new one will need to be created, and any of us can probably do it. But it's up to @devon-sil to decide who should set it up, I think.